ZKLib/Data/UniPoly/Basic.lean

```diff
@@ -41,6 +43,9 @@ def coeffs {R : Type*} (p : UniPoly R) : Array R := p
 variable {R : Type*} [Ring R] [BEq R]
 variable {Q : Type*} [Ring Q]
 
+@[reducible]
+def coeff (p : UniPoly Q) (i : ℕ) : Q := p.getD i 0
```

Comment:
added this definition and used it in existing theorems where possible.

note that mathlib basically _defines_ polynomials as their `coeff` functions, so having the same notion here is useful for conversion